client/src/components/ProfileListings/ProfileCard/ProfileCard.tsx

```diff
@@ -20,41 +21,43 @@ export default function ProfileCard({ profile }: Props): JSX.Element {
 
   return (
     <Card elevation={8}>
-      <CardActionArea>
-        <CardContent className={classes.content}>
-          <Grid container direction="column" alignItems="center" spacing={2}>
-            <Avatar className={classes.avatar} src={`/mock/${profile.photoUrl}`} />
-            <Typography variant="h4">{`${profile.firstName} ${profile.lastName}`}</Typography>
-            <Typography variant="h5">Loving pet sitter</Typography>
-            <Grid item>
-              <Box p={1}>
-                <Rating name="read-only" value={4.5} precision={0.5} readOnly />
+      <Link href={`/profiles/${profile.userId}`} color="inherit" underline="none">
```

Comment:
Please use react-router's Link here and not the one from MUI :)

Reply:
Done